CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+### citus v10.0.2 (March 3, 2021) ###
+
+* Fixes insert query with CTEs/sublinks/subqueries etc
+
+* Removes limits around long table names
+
+* Uses `PROCESS_UTILITY_QUERY` in utility calls
+
+* Preserves colocation with procedures in `alter_distributed_table`
+
+* Grants `SELECT` permission on `citus_tables` view to `public`
+
+* Fixes a bug with `UNION ALL` on PG 13
+
+* Grants read access for columnar metadata tables to unprivileged user
+
+* Prevents using `alter_columnar_table_set` and `alter_columnar_table_reset`
+  on a columnar table not owned by the user
+
+* Fixes a bug due to cross join without target list
+
+* Improves `citus_update_table_statistics` and provides distributed deadlock
+  detection
+
+* Adds a configure flag to enforce security
+
```

Comment:
seems like `Prevent infinite recursion for queries that involve UNION ALL below JOIN` is missing
https://github.com/citusdata/citus/pull/4756

Reply:
I guess we won't include it in 10.0.3, sorry